cinema/views.py

```diff
@@ -76,16 +86,18 @@ def get_serializer_class(self):
         return MovieSerializer
 
 
-class MovieSessionViewSet(viewsets.ModelViewSet):
+class MovieSessionViewSet(mixins.ListModelMixin, mixins.CreateModelMixin, mixins.RetrieveModelMixin,
```

Comment:
It`s correct. But, I think, you should to stat this view Model View Set, because, you don't change behavior when you use all generics mixins